robottelo/ui/locators/base.py

```diff
@@ -1687,8 +1687,9 @@
         By.XPATH, "//td/a[contains(., '%s')]/following::td/div/"
                   "button[contains(@class, 'dropdown-toggle')]"),
     "contentviews.remove_ver": (
-        By.XPATH, ("//td/a[contains(., '%s')]/following::td//"
-                   "a[contains(@ng-click, 'version-deletion')]")),
+        By.XPATH,
+        ("//td/a[contains(., '%s')]/following::td[@class='col-sm-2'][1]"
```

Comment:
Discussed that in IRC. Turns out to be the issue with `following` - it can grab items from other rows (versions) too. Solution is to use `following-sibling` instead.